ui/shared/components/panel/variants/Predictions.jsx

```diff
@@ -29,39 +29,48 @@ const predictionFieldValue = (
 
   const infoValue = predictions[infoField]
 
-  if (dangerThreshold) {
-    value = parseFloat(value).toPrecision(2)
-    let color = 'green'
-    if (value >= dangerThreshold) {
-      color = 'red'
-    } else if (value >= warningThreshold) {
-      color = 'yellow'
-    }
-    return { value, color, infoValue, infoTitle, dangerThreshold, warningThreshold }
+  if (thresholds) {
+    value = parseFloat(value).toPrecision(3)
+    const color = PRED_COLOR_MAP.find(
+      (clr, i) => (thresholds[i - 1] || thresholds[i - 1]) &&
```

Comment:
isn't `(thresholds[i - 1] || thresholds[i - 1])` the same as `thresholds[i - 1]`

Reply:
My fault, it needs to be `(thresholds[i - 1] || thresholds[i ])`